Sources/MockoloFramework/Parsers/SwiftSyntaxExtensions.swift

```diff
@@ -259,7 +259,7 @@ extension IfConfigDeclSyntax {
             in: subModels,
             exclude: [:],
             fullnames: []
-        ).map({ $0 })
+        ).sorted(path: \.value.offset, fallback: \.key)
```

Comment:
Additionally, fix the fragile test caused by https://github.com/uber/mockolo/pull/279 .

Reply:
This operation is same as  https://github.com/uber/mockolo/blob/1fb44ed25e962b52e18f293201a192375f9e018a/Sources/MockoloFramework/Operations/UniqueModelGenerator.swift#L69 .